pkg/config/lighthouse/config.go

```diff
@@ -105,3 +108,55 @@ func (c *Config) Parse() error {
 	}
 	return nil
 }
+
+// InRepoConfig to enable configuration inside the source code of a repository
+type InRepoConfig struct {
+	// Enabled describes whether InRepoConfig is enabled for a given repository. This can
+	// be set globally, per org or per repo using '*', 'org' or 'org/repo' as key. The
+	// narrowest match always takes precedence.
+	Enabled map[string]*bool `json:"enabled,omitempty"`
+	// AllowedClusters is a list of allowed clusternames that can be used for jobs on
+	// a given repo. All clusters that are allowed for the specific repo, its org or
+	// globally can be used.
+	AllowedClusters map[string][]string `json:"allowed_clusters,omitempty"`
```

Comment:
@jstrachan, what is this about? How is the ability to configure the Lighthouse config in the repository tied to a specific cluster?

Reply:
that came from the Prow struct. We don’t use it in any way right now

Reply:
happy to delete it - but there seemed interest in the discussions on this PR of using Prow’s `InRepoConfig` so that the shared config is similar is to prows

Reply:
I see.

Reply:
hmm, similar yes, but if something is not used my initial gut feeling would say not to add it (easier to add something later when we need it opposed to removing something). @eddycharly - wdyt?

Reply:
removed for now. We can easily git revert if we choose to bring it back